Documentation/BabylonCommands.md

```diff
@@ -0,0 +1,50 @@
+# Babylon's Stevenson Commands
+
+This Vapor project consists actually of two projects:
+
+ * The Stevenson framework, aimed to be reused so you can create your own bots interacting with Slack, GitHub, CircleCI and JIRA yourselves
+ * The Stevenson Babylon app, which is the instance of the Bot we use at Babylon, based on the Stevenson framework, and which provides us with the Slack commands we need
+
+# ⚙️ Environment Variables used by our app
+
+The bot instance of Stevenson we use at Babylon uses environment variables to instantiate and configure the various services (see [README.md](../README.md#supported-services)).
+
+See `app.swift` for declaration of those environment variables used in the app, and `configure.swift` for where they are used to instantiate the services.
+
+ENV Var | Description
+--------|--------------
+`SLACK_TOKEN` | Your access token for the Slack API of your team's Slack
```

Comment:
imo it would be better to comment that in code, if the purpose of these environment variables is not clear, not to go out of sync. Readme already mentions environment variables so I'd leave this whole section out.